.github/workflows/windows-latest.yml

```diff
@@ -0,0 +1,60 @@
+name: windows-latest
+
+on:
+  workflow_dispatch:
+  workflow_call:
+  push:
+    branches:
+      - main
+      - release
+    tags-ignore:
+      - v*
+    paths-ignore:
+      - '/*.sh'
+      - '/.*'
+      - '/_*'
+      - '/vcpkg.txt'
+      - 'docs/**'
+      - '**.adoc'
+      - '**.md'
+      - '**.nix'
+      - 'flake.lock'
+      - '.github/workflows/*.yml'
+      - '!.github/workflows/windows-latest.yml'
+      - '!.github/workflows/build.yml'
+      - '!.github/workflows/build-dist.yml'
+  pull_request:
+    paths-ignore:
+      - '/*.sh'
+      - '/.*'
+      - '/_*'
+      - '/vcpkg.txt'
+      - 'docs/**'
+      - '**.adoc'
+      - '**.md'
+      - '**.nix'
+      - 'flake.lock'
+
+jobs:
+  build:
+    uses: ./.github/workflows/build.yml
+    with:
+      os: windows-latest
+
+  build-dist:
+    needs: [build]
+    uses: ./.github/workflows/build-dist.yml
+    secrets:
+      GH_TOKEN: ${{ secrets.GH_PANERON_CI_TOKEN }}
+      # CSC_LINK: ${{ secrets.windows_certs }}
+      # CSC_KEY_PASSWORD: ${{ secrets.windows_certs_password }}
```

Comment:
```suggestion
```

Reply:
This is meant to document the usage of `secrets.windows_certs*`.  I can add more contexts to the lines.

Reply:
Probably we can remove this commented code, right?